src/back/TicketManager/Dockerfile.payment

```diff
@@ -0,0 +1,16 @@
+FROM mcr.microsoft.com/dotnet/sdk:6.0 AS build
+COPY ./src/PaymentService/*.Api/*.csproj /restore/TicketManager.PaymentService.Api/
```

Comment:
Those `*` in folder name will be of a problem in the future for our dear frontend developers, but let's say that it is issue for another day and another person

Reply:
We're gonna force them to install arch linux